apps/admin/frontend/src/components/export_election_package_modal_button.tsx

```diff
@@ -84,8 +82,9 @@ export function ExportElectionPackageModalButton(): JSX.Element {
             <React.Fragment>
               <Button
                 icon="Export"
-                onPress={saveElectionPackageToUsb}
+                onPress={saveElectionPackage}
                 variant="primary"
+                loading={saveElectionPackageToUsbMutation.isLoading}
               >
                 Save
```

Comment:
optional: after watching the video again, a nice-to-have here would be to change the text to `Saving...` or something along those lines while the mutation's running

Reply:
https://github.com/votingworks/vxsuite/assets/530106/77be74e8-2dc3-41a3-985b-3f30f5ca5d39

I like it!